src/modules/tests/test_data/test-data.constants.ts

```diff
@@ -1,4 +1,5 @@
 export enum Competencies {
+  manoeuvres = 'manoeuvres',
```

Comment:
Not sure that manoeuvres should be put here as this maps directly to the competencies in the schema , manoeuvres lives in a different part of the schema